editor/plugins/lightmap_gi_editor_plugin.cpp

```diff
@@ -168,7 +169,9 @@ void LightmapGIEditorPlugin::_bind_methods() {
 LightmapGIEditorPlugin::LightmapGIEditorPlugin() {
 	bake = memnew(Button);
 	bake->set_flat(true);
-	bake->set_icon(EditorNode::get_singleton()->get_gui_base()->get_editor_theme_icon(SNAME("Bake")));
+	// TODO: Rework this as a dedicated toolbar control so we can hook into theme changes and update it
```

Comment:
Why did you add this comment here if this plugin is the same as the other plugins? (e.g. GPUParticlesCollisionSDF3DEditorPlugin)

Reply:
IIRC when I fixed that part I only had two affected plugins in front of me, and I added the same comment to both. Others came with one of the force pushes where it was mostly a blind change, and I didn't think about adding more comments.

Reply:
I can add similar comments all over the place, if that's a concern :) But I think if someone stumbles upon this, they would check if there are other cases (or we would nudge them towards them during the review). If I don't get to these first, that is.

Reply:
> But I think if someone stumbles upon this, they would check if there are other cases

Such comments would make them easier to find, but I guess you'll be the one to fix it anyway.